website/homepage/src/demos/upload_demos.py

```diff
@@ -67,8 +67,10 @@ def upload_demo_to_space(
         )
     return f"https://huggingface.co/spaces/{space_id}"
 
-for category in demos_by_category:
-    for demo in category["demos"]:
-        space_id = "gradio/" + demo["dir"]
-        upload_demo_to_space(demo_name=demo["dir"], space_id=space_id, hf_token=AUTH_TOKEN, gradio_version=gradio_version)
+if __name__ == "__main__":
+    if AUTH_TOKEN is not None:
+        for category in demos_by_category:
```

Comment:
Looks like this includes all of the demos in `recipe_demos.json` but what about demos in other parts of the website like the docs and homepage?